README.md

```diff
@@ -63,6 +67,56 @@ all: {
 }
 ```
 
+### `server`
+
+* Type: `Object`, or a falsy value
+* Default: `false`
+
+When `server` is set to a falsy value, the validator is invoked using `java -jar`, which can be considered normal operation.
+
+Set `server` to an object to start the validator in client mode and connect to an already-running instance of the validator in server mode.
+To start the validator in server mode, use `java -cp "path/to/vnu.jar" nu.validator.servlet.Main <port>`.
+
+```js
+all: {
+  options: {
+    server: {} // connect to a validator instance running in server mode on localhost:8888
+  },
+  src: "app.html"
+}
+```
+
+The `server` object also accepts the `host` and `port` keys, specifying the location of the server.
+
+```js
+all: {
+  options: {
+    server: {
+      host: '192.168.0.5', // your team's local dev tool machine, for example
+      port: 8877
+    }
+  },
+  src: "app.html"
+}
+```
+
+In the following example, a watching task (such as [grunt-contrib-watch][watch]) will complete significantly faster when executing `htmllint:client` as opposed to `htmllint:normal`.
```

Comment:
Or link to https://www.npmjs.com/package/grunt-vnuserver#example and remove the "normal" task from there, that's more confusing than helpful.